### ISSUE TITLE
Tuya Zigbee Curtain module QS-Zigbee-C01

### ISSUE THREAD

**Issue by adbig** [dresden-elektronik/deconz-rest-plugin#3939]
Hello,

I bought a zigbee module to automate my roller shutter garage.
Unfortunally, deconz mistakenly identified it as a TS130F device ( https://es.aliexpress.com/item/4001190275372.html ).
In reallity this is not a switch , it's this :

![IMG_20201127_084837](https://user-images.githubusercontent.com/12624309/102411694-a0f55400-3ff2-11eb-8c90-2ebeeb306ecc.jpg)

Bought here : https://fr.aliexpress.com/item/1005001530266723.html

I successfully manage to go up and down with the detected TS130F configuration but the problem I encounter is that each time I execute the command to go UP or down, the opening/closing time is only 10 secondes. My roller shutter  takes approximatively 30 secondes to Open/close so I have to execute 3 times the opening/closing command.

## Device

- Product name:  Zigbee curtain module
- Manufacturer: _TZ3000_vd43bbfq
- Model identifier: TS130F
- Device type : 
  - Other: Curtain / Roller shutter module

## Screenshots

![basic_cluster_info](https://user-images.githubusercontent.com/12624309/102411660-98048280-3ff2-11eb-84c4-bf1e0613d4c1.jpg)

![node_infos](https://user-images.githubusercontent.com/12624309/102411657-95a22880-3ff2-11eb-97c2-b77c6adf178d.jpg)

thanks !

Adrien

**Comment by Mimiix** [comment]
Hi!

Can you provide all screenshots of the clusters?

**Comment by adbig** [comment]
Hi, 

![group](https://user-images.githubusercontent.com/12624309/102465764-5ef9fb80-404e-11eb-9122-76fb53c5f742.jpg)

![scenes](https://user-images.githubusercontent.com/12624309/102466016-b009ef80-404e-11eb-9573-a10f3be51ef0.jpg)

![windows_covering](https://user-images.githubusercontent.com/12624309/102465812-6caf8100-404e-11eb-9c7c-abcb3171f034.jpg)
![windows_covering_2](https://user-images.githubusercontent.com/12624309/102465819-6de0ae00-404e-11eb-8745-2dc50e9db954.jpg)

**Comment by Smanar** [comment]
For me it s the normal working mode, you probalby need to calibrate the device, you have the command on the cluster 0x0102, and the instructions on your manual.

**Comment by adbig** [comment]
Hi @Smanar  and thank you for your response.

I discovered this thread that is very close to my problem. https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3134

After setting in the XML file the calibration access  to RW I was finally able to calibrate my module !!!!!

```
<attribute-set id="0xf001" description="Tuya Window Covering Setting">
                        <attribute id="0xf001" name="Calibration" type="enum8" default="0" required="m" access="rw">
```
I don't know why this setting is in ReadOnly Mode because it is needed for any new/unconfigured module.

Normally, I use Jeedom to control my domotic, and frankly, it's not very easy to setup this essential feature  ... Do you know if it would be possible to have this kind of feature (calibration) available with the Jeedom DeCONZ plugin ? (I don't know who are the developpers of this plugin).

Thank you for your support !

Adrien

**Comment by Smanar** [comment]
They are already in RW mode on the last version, wich one version are using ?

And I m agree with you, so there is an hidden command (not documented yet) for that

`PUT, PATCH /api/<apikey>/lights/<id> with {"calibration":true}`

**Comment by adbig** [comment]
Hello,

I'm using 2.07.00-raspbian-buster-beta package but I just see there is a new revision 2.08.00-raspbian-buster-beta.

Adrien

**Comment by Smanar** [comment]
Ha yes, the change as done 19 days ago, so it 's recent.

**Comment by adbig** [comment]
SO, it'sgood for me. I close this issue. thank you again for your help !

**Comment by Mimiix** [comment]
@adbig Great:)

**Comment by Sjonnie2018** [comment]
Hi All,

I have this device (TS130F) and have read through all the mentioned threads to find out how I can change the close and open times to more then 10 seconds. Frankly, I am stumped with all the technical info and am looking for some guidance so I can get the module to close/open my roller shutters for 40 seconds.

Could somebody please explain to me in simple words how I can get the device to extend the power when opening or closing my shutters?

Working with Deconz (-gui) version 2.07.01

Thanks a million in advance!

Greetz,

Sjonnie

**Comment by Smanar** [comment]
Do you still have the manual gived with the device ?
I haven't it to make tries, but I think you can do exactly like in the manual except you use deconz GUI to set the "set calibration mode" instead of the app.

**Comment by Sjonnie2018** [comment]
Hi Smanar,

Thanks for your reply and support!

I have the manual but it does not give any information about changing the timings for the module. I can not find "set calibration mode" in deconz-gui. Maybe I overlooked. If so, please tell me where I can find it.

Greetz,

Sjonnie

**Comment by adbig** [comment]
Hi,

First, you must have the latest version of Deconz because in the previous versions I had to modify the "general.xml" file to allow the switch to calibration mode.
This section was in read only mode : 
`                        <attribute id="0xf001" name="Calibration" type="enum8" default="0" required="m" access="rw">
`

At the the bottom of this screen you'll see "Calibration : end"

![](https://user-images.githubusercontent.com/12624309/102465819-6de0ae00-404e-11eb-8745-2dc50e9db954.jpg)

- First ensure you curtain is fully closed.
- Set the calibration  to "start" and click Write. Now you are in Calibration mode.
- Above, click on the Exec button in the function  Up / Open. In the calibration mode the 30 secondes duration is disable.
- Once your curtain is fully open click on the "Exec" button in the function "Stop"
- Click on the Exec button in the function Down / Close
- Once your curtain is fully closed click on the "Exec" button in the function "Stop"
- Set the calibration  to "end" and click Write

Enjoy, your module is calibrated !

(sorry for appromative my english ...)

Adrien

**Comment by Sjonnie2018** [comment]
@adbig : You Sir, are now officially declared Sjonnie's Hero of The Day 2021! :)

Very clear explanation and instructions. Calibrating the module was very easy following your guidance.

Now I can safely order these for my other roller shutters.

One note for future generations: after the last "write" action the device shows "start" in the calibration field. You can remedie that by doing a read of the attributes.

Again Sir: Kudos!

**Comment by adbig** [comment]
Glad it helped you !

**Comment by incognitomailincognitomail** [comment]
@adbig can you explain how to connect to curtain module, I didn't find calibration dialogs in DeconZ plugin
And how I can edit general.xml file

**Comment by Sjonnie2018** [comment]
You will have to open deconz-gui (in your case: connect to the device in the last screenshot). After that you will find the node in the gui. Select the right most "radio button" on the top right of the node and check the cluster info.

**Comment by incognitomailincognitomail** [comment]
@Sjonnie2018 What is Conbee (I have ZigBee cc2531 usb dongle).
Can you please send me toturial

**Comment by Sjonnie2018** [comment]
Hi, I figured from your screenshot that you had a Conbee (2) stick from Dresden Electronics. I haven't got a clue how to connect Deconz(-GUI) to your cc2531 stick. Sorry! I don't quite understand how and why you would use Deconz if you do not have a device from Dresden Electronics. Maybe I am missing something or do not fully understand you situation. If so, please enlighten me.

The tutorial I used is the one adlbg wrote down in post 8 of this thread.

**Comment by Mimiix** [comment]
@incognitomailincognitomail Deconz only work with Conbee / Raspbee devices created by the vendor.

**Comment by incognitomailincognitomail** [comment]
@Mimiix thanks.
I can use Conbee stick to calibrate the curtain module only, and after the calibration to use my ZigBee cc2531 usb dongle)

**Comment by Sjonnie2018** [comment]
I am a bit stumped here. You have a Conbee II stick to configure the roller shutter module and after that you want to use your cc2531 to control it? If that's the case the information in this particular thread will help you achieve your goal. Connect your Conbee II stick to the device running Deconz (using a USB extension cord) and follow the steps in this thread. Good luck!

**Comment by Mimiix** [comment]
> I am a bit stumped here. You have a Conbee II stick to configure the roller shutter module and after that you want to use your cc2531 to control it? If that's the case the information in this particular thread will help you achieve your goal. Connect your Conbee II stick to the device running Deconz (using a USB extension cord) and follow the steps in this thread. Good luck!

I am wondering if that would work. To pair it to the cc stick he would need to reset the device. Wondering if the device keeps the settings.

**Comment by Sjonnie2018** [comment]
I honestly don't have a clue! I wouldn't even bother if I was him and just buy a Conbee II. Not overly expensive and as a bonus he gets support from this community :)

**Comment by Smanar** [comment]
And for information, if It work, it s possible use the API, if you don't have the GUI
`PUT, PATCH /api/<apikey>/lights/<id> with {"calibration":true}`

**Comment by incognitomailincognitomail** [comment]
ok thanks for all.
Can I calibrate the roller shutter in other way? (without Conbee)

**Comment by adbig** [comment]
With the tuya smart life app .... If you Read the doc provided with your module it's explained...

**Comment by incognitomailincognitomail** [comment]
@adbig can you please send link with the details
Additional Q,
With smart life I can use my ZigBee cc2531 usb dongle?

**Comment by Mimiix** [comment]
@incognitomailincognitomail This repository is for deCONZ only. This issue is about the specific device for deCONZ. Keep it on topic here. With teh CC Stick, your kind of on your own.

@adbig Does the device work properly with deCONZ?

**Comment by adbig** [comment]
Yes the device works very Fine with decone since monthes.

**Comment by incognitomailincognitomail** [comment]
Hi 

I don't see all module only "hone Automation configuration tool"
@adbig Please advise 

![image](https://user-images.githubusercontent.com/88643642/130649281-1fbade54-a4ba-4007-a424-624427765004.png)

**Comment by Smanar** [comment]
Your devices are not included at all.
And as your network is new, can be a deconz issue, start from the start, check in phoscon if there is a firmware number, test with USB extension, check logs, checks config, .....

**Comment by 2chilled** [comment]
Hi, using REST API I can activate calibration mode using:
`curl "http://openhabian:80/api/abc/lights/22" -d '{"calibration":"Start"}' -XPUT`

But how can I deactivate it? I've tried setting calibration to "End", but that does not have any effect.

**Comment by Smanar** [comment]
"calibration" work with true and false.

**Comment by 2chilled** [comment]
@Smanar I tried with true and false, but it seems to be still in calibration mode after putting false. This is the sequence I tried:
```
curl "http://openhabian:80/api/abc/lights/22" -d '{"calibration":true}' -XPUT
curl "http://openhabian:80/api/abc/lights/22/state" -d '{"lift":0}' -XPUT
curl "http://openhabian:80/api/abc/lights/22/state" -d '{"lift":"stop"}' -XPUT
curl "http://openhabian:80/api/abc/lights/22/state" -d '{"lift":100}' -XPUT
curl "http://openhabian:80/api/abc/lights/22/state" -d '{"lift":"stop"}' -XPUT
curl "http://openhabian:80/api/abc/lights/22" -d '{"calibration":false}' -XPUT
```
The stop commands were issued when the curtain was fully open or closed. After that sequence when setting "lift" to 50, the curtain always goes to end position. What am I missing?

**Comment by AdrienBigot** [comment]
@Smanar are you sure it is start and stop  ?
In the XML file I read : 
```
     </attribute-set>
                <attribute-set id="0xf001" description="Tuya Window Covering Setting">
                        <attribute id="0xf001" name="Calibration" type="enum8" default="0" required="m" access="rw">
                                <value name="Start" value="0"></value>
                                <value name="End" value="1"></value>
                        </attribute>
                        <attribute id="0xf002" name="Motor Reversal" type="enum8" default="0" required="m" access="rw">
                                <value name="Off" value="0"></value>
                                <value name="On" value="1"></value>
                        </attribute>
        </attribute-set>

@2chilled why don't you try with Deconz GUI ?

**Comment by mahe-ymc** [comment]
@AdrienBigot I'm running on a headless raspi without any X libs installed. Do you think I'd have more luck running the GUI version and connecting via vnc?

**Comment by 2chilled** [comment]
@AdrienBigot I'm running Conbee2 on a headless raspi without any X libs installed. If calibration can't be done with REST API I'll bite the bullet and change that. I think I need to connect via vnc then?

**Comment by Smanar** [comment]
Sure, from the code

```
        qint64 value = 0x00;
        if (map["calibration"].toBool())
        {
            value = 0x01;
        }
```

So the code send the value 0 or 1 according to True or False (true = 0x01 = End and False = 0x00 = Start, and BTW they are reversed .....)
You have not an error message as return when you make the request ?

**Comment by 2chilled** [comment]
@Smanar No, I'm getting 200 response code. I guess your `toBool` ist not very strict with it's inputs ;)

Ok, does that mean I have to activate calibration with `false` and deactivate it with `true`?

**Comment by 2chilled** [comment]
Just tried with switching false to true and vice versa. No change. Device stays in calibration mode as it does not stop moving. Power off and power on leads to the initial state, that is 10 seconds duration until stop. Do I really need to use the GUI for calibrating?

**Comment by Smanar** [comment]
You have a 200 code, but there is a message too ?
IDK, I have checked the code again, and all seem fine, you haven't access to log ?

**Comment by AdrienBigot** [comment]
@2chilled I have the same setup as you with a headless Debian buster on a raspberry Pi2 and I use deCONZ gui over ssh.

On the machine running deCONZ : 
- Ensure you have X11forwarding to yes in your /etc/ssh/sshd_config 
- Stop deCONZ deconz service first :  systemctl stop deconz.service

On your client : 
ssh -X user@your_server_address.
Once you're logged, just launch deCONZ binary (/usr/bin/deCONZ for me). It opens a new windows with your deCONZ GUI :-D   (don't try with root user. On my side I must ssh with my "pi" user (who usually also run deconz.service).

Adrien

**Comment by 2chilled** [comment]
@AdrienBigot Thank you very much for that howto! Just realized that I already have all tools in front of my eyes, but I didn't see them. X11Forwarding, that's the key :) My rollerblind works now and calibrating is easy peasy using the GUI. Thanks again, appreciate it :)

**Comment by AdrienBigot** [comment]
Happy it works for you !

But it's strange that we cannot do the calibration with API. I admit I never try do do it in this way and don't want to reset my working config :-D
I know somebody that will buy this device. I'll try to test the calibration with API on my side and will give a feedback here.

Adrien.

**Comment by 2chilled** [comment]
I still have a bunch of these devices to install, I'll try to post API Info requested by @Smanar this evening.

**Comment by Smanar** [comment]
But I realy don't understand, all seem fine, can you take a look in log in same time, you are making the request (with aps/info/info_l2)

The synthax is good because it work to enable it, but it s like `if (map["calibration"].toBool())` is never trigger, so the value is always 0x00

**Comment by 2chilled** [comment]
I can have a look tomorrow, but where is that log located? I see nothing in journal of deconz systemd unit.

**Comment by Smanar** [comment]
If you have headless mode, you have only 1 solution, use command line https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/deCONZ-debug-flags

**Comment by 2chilled** [comment]
I have a new issue with this device to report. Sometimes, I don't have a pattern yet, I cannot write the state of the device. I provide a communication flow:
```
curl "http://192.168.0.207:80/api/2D58761B15/lights/51/state" -XPUT -d'{"lift": 10}'
[{"success":{"/lights/51/state/lift":10}}]

curl "http://192.168.0.207:80/api/2D58761B15/lights/51" | json_pp
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   364  100   364    0     0  72800      0 --:--:-- --:--:-- --:--:-- 72800
{
   "etag" : "95ff636c780d8c1d6751efc547f0b537",
   "hascolor" : false,
   "lastannounced" : null,
   "lastseen" : "2021-10-04T07:38Z",
   "manufacturername" : "_TZ3000_vd43bbfq",
   "modelid" : "TS130F",
   "name" : "Rolladen Wohnzimmer Süd OG 1",
   "state" : {
      "bri" : 0,
      "lift" : 0,
      "on" : false,
      "open" : true,
      "reachable" : true
   },
   "swversion" : null,
   "type" : "Window covering device",
   "uniqueid" : "cc:86:ec:ff:fe:be:71:0a-01"
}
```

When I hit the manual switch and stop it again, the device is writeable again through Zigbee. Please tell me what info you need in addition to tackle this. Thank you!

**Comment by Smanar** [comment]
I m not sure to understand the issue.
You are making the request `curl "http://192.168.0.207:80/api/2D58761B15/lights/51/state" -XPUT -d'{"lift": 10}'`
You have the return "success" but the device don't move and the json stay at lift=0 ?

**Comment by 2chilled** [comment]
@Smanar Yes, you understood correctly. This problem occurs with 2 devices installed on top of each other. I also tested them using deconz GUI and I'm getting timeouts there. What I also observed in the GUI is that both devices have bad connection and somehow they're not connecting to the nearest Zigbee routers around them. For example I have a plug 1m away from them which is very well connected but they just don't connect to it. I guess I will pull them out the wall and check if it helps or really replace them in the end.

For the deconz-rest-plugin the isolated issue I'm observing that it delivers false positives. It shouldn't return success if it wasn't successful. But it does in my case.

**Comment by 2chilled** [comment]
@Smanar Update: I removed both devices from deconz and added them again. After that everything works as it should :) But the last point from above remains in general.

**Comment by Smanar** [comment]
Just for information about the deconz working mode. When you make a request, you will have a "success" if deconz can send the request, it don't mean the device will use it.

- You make the http request
- Deconz send the zigbee request
- You have the "success" message
- The device execute the request
- The device send report
- deconz Update the value

So bad connexion can explain your issue, but the question is why you have bad connexion.

**Comment by 2chilled** [comment]
Ok, then deconz behaves like expected. Yeah, that one still remains. At least I know a workaround for next time ;)

**Comment by 69franky** [comment]
> @69franky Your method to terminate calibration mode worked for me for some of my qs-zigbee-c01 devices but not for all of them. Maybe it also depends on actual manufacturer. I have both "_TZ3000_vd43bbfq" and "_TZ3000_fccpjz5z" installed.

I am evaluating only one device, which is from  "_TZ3000_fccpjz5z", and I could calibrate it. All this does not sound like a sustainable solution. Keep you posted.

**Comment by 2chilled** [comment]
@Smanar My repeater is in place for 2 days now and the problem still persists. It seems like these devices have problems with connecting with my other routers. There are 3 devices suffering from the issue described above and all of them have no connection to the Conbee2 coordinator. Is there anything I can do about it? Do you know of an alternative rollershutter device which also supports the lift API?

**Comment by Smanar** [comment]
@qba667 you have tried the code modification ?

**Comment by AoSpades** [comment]
Hi there! I've got a similar device which I'm trying to use over Zigbee2Tasmota and the console. 

Is there any way to enter calibration mode using a ZbSend command on these devices?

**Comment by Mimiix** [comment]
@AoSpades thats kinda off-topic..

**Comment by AoSpades** [comment]
@Mimiix As a noob I'm out of wits here, so any help would be appreciated.

**Comment by Mimiix** [comment]
@AoSpades ITs simple: Zigbee2Tasmota is not deCONZ, thus offtopic 😅

**Comment by Smanar** [comment]
Hello, someone here have issue with this device ignoring some request ?
Was like it was disconnecting, but work again after a time ?

**Comment by 2chilled** [comment]
Yep, I have two devices which behave sort of like that. Both have no direct coordinator connection. They can work fine for a few weeks and suddenly ignore commands for some days. Then it repeats. But they are always able to send data to coordinator. I know that because when in bad state and pushing the manual button they always tell coordinator their curtain position.

**Comment by Smanar** [comment]
Ok, thx.
Because I have another user with 10 coverings, and lot of them stop working randomly, can work naturaly after some time or with some request using the GUI.

And I don't found similars issues on google, idk if it s from the device or if we can do something ....

**Comment by AdrienBigot** [comment]
Yes !!! I'm in that case.
I've put in place an alarm in HomeAssistant to send me a SMS when my garage door is unavailable because I suspected this problem.
And sometimes, (I can't reproduce it) I don't know why it becomes unavailable and it is back a few minutes later. First, I thought it was a conflit with my wifi, I try to change channel , try to add a 2 meters USB extension cable to place my conbee II away , but nothing change.
I've tried to put some debug log in deCONZ, but my eyes has burned. So I left it as is.  I'm pleased to hear about that because I always thought the problem was on my side.

**Comment by Smanar** [comment]
Thx I will ask too on Z2m issue to see if it s same for them.

**Comment by 2chilled** [comment]
For them it seems even buggier as they also have problems when these beasts are directly connected to coordinator. I have 7 qs-zigbee-c01 devices running and five of them which have direct connection to conbee 2 are working without any issues for months now.

**Comment by Smanar** [comment]
So they work better with direct connexion to coordinator for you ?

**Comment by 2chilled** [comment]
Yes, they just work when directly connected to coordinator. But I have to add that (unfortunately) all these five directly connected devices have other ZigBee routers with proper antennas around them in a 2-3cm margin. The nearest proper routers around the two black sheeps have a distance of ~50cm. Maybe that's the actual root cause, don't know.

**Comment by Sjonnie2018** [comment]
If I understand correctly one would be best of with a more powerful coordinator. So maybe a Conbee 3 with more transmitting power would solve the problem?

**Comment by 2chilled** [comment]
Yeah, perhaps. But that ConBee 3 probably would have a too high energy consumption for itself. Maybe someone can make a test and place a good ZigBee router very near to problematic qs-zigbee-c01 to see if that makes it work reliable. E.g. that Ikea "repeater" would be a good fit for that. I'll do it by myself before deciding to replace them with Shellys ;)

**Comment by Smanar** [comment]
It's perhaps possible to "fixe a route" https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Source-Routing#fixed-source-routes-source to force the device use the coordinator

**Comment by Sjonnie2018** [comment]
*snip*I'll do it by myself before deciding to replace them with Shellys ;)

Well... there is an alternative I hadn't thought about! The Shelly 2.5 is dirt cheap and would be an alternative to my ZigBee roller shutter devices. Shelly has a Domoticz integration (have to further investigate that) and that would make it a great companion to my home automation setup.

Thank you very much for this tip. Much appreciated!

**Comment by 2chilled** [comment]
> It's perhaps possible to "fixe a route" https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Source-Routing#fixed-source-routes-source to force the device use the coordinator
> 
> But if I remember, it s totaly random for @Sjonnie2018

I already tried source routing but unfortunately it made things worse for me. The whole network became much less responsive resulting in command timeouts. I guess that feature is just not yet mature enough.

@Sjonnie2018 Yeah, that's true. Only reason I didn't replace them yet is that IMO ZigBee is by far the better network protocol for home automation devices. By design it eliminates a whole bunch of security, privacy and open api problems you have with other technologies like WIFI. There is just no way for devices to talk to China Cloud because there is no gateway to the outside of your house network. Admittedly Shelly is an exception here, but it's still WIFI ;)

**Comment by Sjonnie2018** [comment]
> @Sjonnie2018 Yeah, that's true. Only reason I didn't replace them yet is that IMO ZigBee is by far the better network protocol for home automation devices. By design it eliminates a whole bunch of security, privacy and open api problems you have with other technologies like WIFI. There is just no way for devices to talk to China Cloud because there is no gateway to the outside of your house network. Admittedly Shelly is an exception here, but it's still WIFI ;)

Hi,

I think I can manage the communication of the device to the outside world. I have separate SSID's/VLAN's for my talkative devices. And my UniFi router is very capable of blocking any communication over the Internet. But it's sound advice!

I have just ordered one Shelly 2.5 to experiment with. I have one ZigBee roller shutter device that has more than his fair share of failing so I will replace that one and see if I can manage it in Domoticz and test the reliability.

(They really are dirt cheap so experimenting is a joy :-) )

Again thanks for your invaluable tip! I hadn't thought of it :-)

**Comment by kjellvanzon** [comment]
I have 2 sunshades that I operate with 2 of these devices. Calibration works and I can do it via the Deconz interface. Only after calibration do the shutters no longer open fully. I think this has to do with the fact that going "up" takes more power or something. After each closing, the sunshade opens a little less. I've already calibrated about 5 times. Also tried increasing the opening time by pressing the stop button a few seconds later while calibrating, but that doesn't work either. Anyone an idea how to solve this?

**Comment by belly-89** [comment]
> I don't remember but I have this device, and if you have the GUI you have a "Calibration time (tenth of a second)" in the covering cluster, just increase this value.

Thanks! I found the "Calibration time (tenth of a second)" in de GUI. On both sunscreens it has a different number. Do I need to change it before calibration or after to make an correction? Sorry, not really sure what to do!

**Comment by Smanar** [comment]
On mine there is no calibration, it s just a timed mouvement, so I put the value I want.
You can try  a calibration, read the value, add 20% and save it on the device.

**Comment by belly-89** [comment]
> On mine there is no calibration, it s just a timed mouvement, so I put the value I want. You can try a calibration, read the value, add 20% and save it on the device.

So it is not the time of the calibration but the time the screen is active. Will try it tonight! Thanks

**Comment by Smanar** [comment]
Yep the time the contact stay active when you send an order.

**Comment by belly-89** [comment]
Unfortunately, that doesn't work for me. The time that it is active is indeed longer. But if I close my awning to 80% and then open it again, it still won't close. I think this is because closing is faster than opening because it takes less energy. So in the end it just keeps getting worse and worse. 1 sunscreen stops automatically at 100%, so I can ensure that I always open or close it completely. I don't want to open my 2nd screen 100% because of saftey reasons, so this is not an option. anyone got an other idea?

**Comment by Smanar** [comment]
I think you can't have better with this device, it don't support "lift feature", it's jut a fake one using timing, and as you say, it take more time to open the covering than to close it.

**Comment by belly-89** [comment]
Thanks! Then 

> I think you can't have better with this device, it don't support "lift feature", it's jut a fake one using timing, and as you say, it take more time to open the covering than to close it.

I think you are correct. Do you now an alternative that works with Deconz and has an lift feature?

**Comment by Smanar** [comment]
Yes, but the problem is there are never module, but it's the covering itself that support this feature.
It s inside the covering they have electronic to know exactly the cover position.

**Comment by KipiKland** [comment]
been delivered the new module SC500ZB-V2 seen again as TS130F by deCONZ.  working as a charm

![image](https://user-images.githubusercontent.com/63462251/176500357-76b7ce03-b900-49e9-b228-18a7ac8cad9c.png)

Strange that it appears as a light on phoscon webapp
![image](https://user-images.githubusercontent.com/63462251/176500494-d907bfc9-0901-40e7-8bfd-8cba228ba4c3.png)

![image](https://user-images.githubusercontent.com/63462251/176500809-af21a068-4302-4133-9ac7-815d76257856.png)

![image](https://user-images.githubusercontent.com/63462251/176500934-79ac7a36-6fa2-4b18-b283-193078ce83ce.png)
